src/lines/fmtp-line.ts

```diff
@@ -17,6 +17,36 @@
 import { NUM, REST } from '../regex-helpers';
 import { Line } from './line';
 
+/**
+ * Parse the fmtpParams because SDP has no such util function for that.
+ *
+ * @param fmtpParams -  like `key1=val1;key2=val2;key3=val3".
+ * @returns A JS key-value object.
+ */
+export function parseFmtpParams(fmtpParams: string) {
```

Comment:
Does this need to be exported?  At this point I think it's just an internal helper function, right?

Reply:
I see you pass a map in down below, I think we still don't need to export this.  This will be used for the 'internal' parsing (where it takes the sdp string and parses it).  The map constructor will be used when someone wants to craft a line on their own, so they'll just add the params specifically to the map.

Reply:
Ah I see, I didn't think there was a need for parsing those after this, but I didn't realize RTCRtpCodecParameters still holds them in that format.